src/protagonist/DLCS.HydraModel/Converters/ImageDeliveryChannelsConverter.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace DLCS.HydraModel.Converters;
+
+public class ImageDeliveryChannelsConverter : JsonConverter
```

Comment:
You could make this `: JsonConverter<DeliveryChannel[]>` as you know the concrete type. Rather than getting an `object` you'll get `DeliveryChannel[]?` and won't need to implement `CanConvert()`